tf2_ros/src/tf2_ros/buffer.py

```diff
@@ -51,7 +51,7 @@ class Buffer(tf2.BufferCore, tf2_ros.BufferInterface):
     known frames. 
     """
 
-    def __init__(self, cache_time = None):
+    def __init__(self, cache_time = None, debug = False, node = None):
```

Comment:
Since both new arguments can only be used together how about just passing `node` conditionally when debugging is desired?

Please update the docblock of the constructor accordingly.